core/templates/pages/admin-page/roles-tab/admin-roles-tab.directive.ts

```diff
@@ -142,9 +142,9 @@ angular.module('oppia').directive('adminRolesTab', [
             formResponse.category, formResponse.username,
             formResponse.languageCode
           ).then(() => {
-            ctrl.setStatusMessage(
-              'Successfully added "' + formResponse.username + '" as ' +
-              formResponse.category + ' reviewer.');
+            ctrl.setStatusMessage(              
```

Comment:
No trailing spaces! (I was expecting pre-push lint checks to catch this, can you confirm that pre-push checks are working for you?)

Reply:
this is resolved I guess. (No it didn't work for me)